docs/tutorials/sharded-computation.md

```diff
@@ -154,21 +181,26 @@ jax.debug.visualize_array_sharding(result)
 print(result)
 ```
 
++++ {"id": "4a195c780f22"}
+
 This gives you a function with the particular output sharding you'd like.
 
+## 3. Manual parallelism with `shard_map`
 
+In the automatic parallelism methods explored above, you can write a function as if you're operating on the full dataset, and `jit` will split that computation across multiple devices. By contrast, with {func}`jax.experimental.shard_map.shard_map` you write the function that will handle a single shard of data, and `shard_map` will construct the full function.
```

Comment:
Use ```{func}`~jax.experimental.shard_map.shard_map` ``` to avoid spelling out the whole import path in the rendered version.

Reply:
![image](https://github.com/google/jax/assets/19637339/ac628660-55a2-4f15-805c-f8f2a7db5e5f)

No autolinking after using `~` here @jakevdp